adcm_client/wrappers/api.py

```diff
@@ -124,6 +125,10 @@ def auth(self, username, password):
         self.schema = self.client.get("{}{}schema/".format(self.url, self.api_url))
         self.objects = self._parse_schema(self.schema, is_allure=IS_ALLURE)
         self.api_token = token['token']
+        try:
+            self.api_version = self.objects.info.list()['adcm_version']
```

Comment:
I don't like this. You take ADCM_version and store this like API_version. Why?

Reply:
Because we have _api, api_url, api_token. I also couldn’t just name "version", it would cause a conflict with the Bundle and Prototype objects.

Reply:
Why don't you use adcm_version?

Reply:
Ok, but I don't like this. We have a client for API, and I think ADCM is a complete application.